extensions/wikia/AdEngine/js/template/floor.js

```diff
@@ -109,6 +109,13 @@ define('ext.wikia.adEngine.template.floor', [
 		}
 	}
 
+	adContext.addCallback(function () {
```

Comment:
I wonder if we shouldn't rename the `addCallback` method to `addMercuryCallback` since it's being used only in Mercury... But this is not anything to change at this point (in this PR) -- just a thought for future :japanese_ogre: